addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2704,20 +2704,17 @@ options.registry.CoverProperties = options.Class.extend({
     //--------------------------------------------------------------------------
 
     /**
-     * Handles a background change.
+     * Toggles background image on or off.
      *
      * @see this.selectClass for parameters
      */
-    background: async function (previewMode, widgetValue, params) {
-        if (widgetValue === '') {
+    toggleBgImage: async function (previewMode, widgetValue, params) {
+        if (widgetValue) {
+            // TODO: use trigger instead of el.click when possible
```

Comment:
This is intentional. The `toggleBgImage` that replaces the old `background` method receives two possible values for `widgetValue`: empty string when disabling the image, "true" string when enabling the image.
Previously `background` received the selected image's URL from the image picker.

Reply:
I don't understand how this is related to the todo :thinking: Can't you use `.enable()` on the widget instead of `.click()` ? Still ugly and to refactor but this is the accepted way to do it right now and it would be one less TODO in the code :)